### ISSUE TITLE
Mastery of the Crystals. Quest is bugged

### ISSUE THREAD

**Issue by WindWarrior75** [dalaranwow/dalaran-wow#5711]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

Current behaviour: The three crystals you are provided with do not cast their spells as intended.

Expected behaviour: The three crystals should cast their spells when near Frenzyhearts

Steps to reproduce the problem:

1. Accept "Mastery of the Crystals" quest
2. Go to Sparktouched Haven and collect crystals from a chest
3. Place crystals near Frenzyheart (which does not cast their spells)

Include proofs for this behaviour
![WoWScrnShot_040219_082106](https://user-images.githubusercontent.com/47932086/55402419-72370180-5521-11e9-8b1a-1f09ff323859.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

The quest is Mastery of the Crystals
The items are "Crystal of the Frozen Grip", "Crystal of Unstable Energy", and "Crystal of the Violent Storm".

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
#2583
